pandas/core/groupby/groupby.py

```diff
@@ -2675,6 +2674,51 @@ def rank(
         Returns
         -------
         DataFrame with ranking of values within each group
+
+        See Also
+        --------
+        Series.groupby : Apply a function groupby to a Series.
```

Comment:
I know that other groupby docs include Series/DataFrame .groupby in the See Also, but IMO they're not helpful (especially since they don't link to anything).

Reply:
Have opened #42406 for this